examples/prescriptors/neat/prescribe.py

```diff
@@ -0,0 +1,194 @@
+                    df_dict[ip_col].append(prescribed_ip)
+
+            # Save prescriptions so far to file
+            pres_df = pd.DataFrame(df_dict)
+            pres_df.to_csv(TMP_PRESCRIPTION_FILE)
+
+            # Make prediction given prescription for all countries
+            pred_df = get_predictions(start_date_str, date_str, TMP_PRESCRIPTION_FILE)
+
+            # Update past data with new day of prescriptions and predictions
+            pres_df['GeoID'] = pres_df['CountryName'] + '__' + pres_df['RegionName'].astype(str)
+            pred_df['GeoID'] = pred_df['CountryName'] + '__' + pred_df['RegionName'].astype(str)
+            new_pres_df = pres_df[pres_df['Date'] == date_str]
+            new_pred_df = pred_df[pred_df['Date'] == date_str]
+            for geo in df['GeoID'].unique():
+                geo_pres = new_pres_df[new_pres_df['GeoID'] == geo]
+                geo_pred = new_pred_df[new_pred_df['GeoID'] == geo]
+                for ip_col in IP_COLS:
+                    eval_past_ips[geo] = np.append(eval_past_ips[geo],
+                                                   geo_pres[ip_col].values[0])
+
+                # It is possible that the predictor does not return values for some regions.
+                # To make sure we generate full prescriptions, this script continues anyway.
+                # Geos that are ignored in this way by the predictor, will not be used in
+                # quantitative evaluation. A list of such geos can be found in unused_geos.txt.
+                if len(geo_pred) != 0:
+                    eval_past_cases[geo] = np.append(eval_past_cases[geo],
+                                                     geo_pred[PRED_CASES_COL].values[0])
+
+        # Add prescription df to list of all prescriptions for this submission
+        pres_df['PrescriptionIndex'] = prescription_idx
+        prescription_dfs.append(pres_df)
+
+    # Combine dfs for all prescriptions into a single df for the submission
+    prescription_df = pd.concat(prescription_dfs)
+
+    # Create the output path
+    os.makedirs(os.path.dirname(output_file_path), exist_ok=True)
+
+    # Save to a csv file
+    prescription_df.to_csv(output_file_path, index=False)
+    print('Prescriptions saved to', output_file_path)
+
+    return
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-s", "--start_date",
+                        dest="start_date",
+                        type=str,
+                        required=True,
+                        help="Start date from which to prescribe, included, as YYYY-MM-DD. For example 2020-08-01")
+    parser.add_argument("-e", "--end_date",
+                        dest="end_date",
+                        type=str,
+                        required=True,
+                        help="End date for the last prescription, included, as YYYY-MM-DD. For example 2020-08-31")
+    parser.add_argument("-p", "--prior_ips",
```

Comment:
@ofrancon I've changed all `prescribe.py` scripts to use `-ip`, and `--interventions_past` instead of `--interventions_plan`, since these are past ips, whereas the predictor uses future ips.

Reply:
Good point